dspace-jspui/src/main/webapp/dspace-admin/list-formats.jsp

```diff
@@ -30,14 +29,16 @@
 <%@ page import="org.dspace.core.Context"%>
 <%@ page import="org.dspace.app.webui.util.UIUtil"%>
 <%@ page import="java.util.List" %>
+<%@ page import="java.lang.StringBuilder" %>
 <%@ page import="org.dspace.content.service.BitstreamFormatService" %>
 <%@ page import="org.dspace.content.factory.ContentServiceFactory" %>
 
 
 <%
-    BitstreamFormatService bitstreamFormatService = ContentServiceFactory.getInstance().getBitstreamFormatService();
-    List<BitstreamFormat> formats =
-        (List<BitstreamFormat>) request.getAttribute("formats");
```

Comment:
This should not stop us from merging, but the attribute "formats" is still set: https://github.com/DSpace/DSpace/blob/b6875cddaa42602bfa61c2521c57979b5c640159/dspace-jspui/src/main/java/org/dspace/app/webui/servlet/admin/BitstreamFormatRegistry.java#L171
Perhaps we should change the BitstreamFormatRegistry and also the other JSP (see https://github.com/DSpace/DSpace/blob/b6875cddaa42602bfa61c2521c57979b5c640159/dspace-jspui/src/main/java/org/dspace/app/webui/servlet/admin/BitstreamFormatRegistry.java#L136) as well?